DataFormats/Math/src/classes.h

```diff
@@ -225,9 +224,6 @@ namespace DataFormats_Math {
     ROOT::Math::MatRepStd<double, 10 , 10> smdcw;
     ROOT::Math::MatRepStd<double, 2 , 3> smdcw1;
 
-    //Used by MET Significance matrix
-    ROOT::Math::SMatrix<double,2> smat;
```

Comment:
OK, I backed out this change and pushed to this branch again.  This change is no longer in this PR.